### ISSUE TITLE
Enhance Speech Viewer to Let User Adjust Font Size

### ISSUE THREAD

**Issue by jhomme** [nvaccess/nvda#8388]
### Steps to reproduce:

Open Speech Viewer

This dialog has no ability to adjust font size to make the window easier to see for people who are older or who have low vision. It would be good to be able to adjust the size and possibly font face.

### Expected behavior:

Speech Viewer would have a font size adjustment control.

### Actual behavior:

The user is forced to use the default font size and font face.

### System configuration:
NVDA version: 2018.1.1
NVDA Installed or portable: Installed

Other information:
Windows version: 10

Name and version of other software in use when reproducing the issue:

Firefox 60.2

### Other questions:

Does the issue still occur after restarting your PC? Yes

Have you tried any other versions of NVDA? Yes

This is really a feature request, so this would not apply.

**Comment by Adriani90** [comment]
Closing as duplicate of #8128.

**Comment by hwf1324** [comment]
Obviously this issue is different from #8128.

The request in this issue is to be able to adjust the font and size of the text in the speech viewer, instead of only displaying the last text read aloud.

So this issue should be reopened.

**Comment by Adriani90** [comment]
Thanks for pointing out, I am reopening.

**Comment by hwf1324** [comment]
If anyone is interested, try this [try build](https://dl.nvdacn.com/GuestUpload/cb8a1f4adaa158155dec066a64351592.exe
) version.

In the speech viewer:
I added a checkbox to show only the last speech.
Also added a button to adjust the font style.

**Comment by coreymoore** [comment]
I would like to bump this to bring it to the developers' attention. I use the speech viewer with the text size increased as an assistive tool but I only need to use the screen reader for reading large blocks of text (helps me with visual tracking line by line, which can be challenging to me because of nystagmus) and accessibility testing, so I am constantly closing and opening NVDA throughout my day (10+ times some days). Having to readjust the text size every time I reopen the screen reader is challenging, especially because the only method to resize is to use the scroll wheel and it is really easy to overshoot the size I need.

**Comment by hwf1324** [comment]
> I would like to bump this to bring it to the developers' attention. I use the speech viewer with the text size increased as an assistive tool but I only need to use the screen reader for reading large blocks of text (helps me with visual tracking line by line, which can be challenging to me because of nystagmus) and accessibility testing, so I am constantly closing and opening NVDA throughout my day (10+ times some days). Having to readjust the text size every time I reopen the screen reader is challenging, especially because the only method to resize is to use the scroll wheel and it is really easy to overshoot the size I need.

You can try my build, although it's a bit of ugly code. Also note that it changes the configuration file, just in case please make a backup.
